Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -5033,36 +5033,38 @@ def return_results(results):
         demisto.results(None)
         return
 
-    if results and isinstance(results, list) and len(results) > 0 and isinstance(results[0], CommandResults):
+    elif results and isinstance(results, list) and len(results) > 0:
```

Comment:
if the first two conditions are True then the third must be True.
```suggestion
    elif results and isinstance(results, list):
```

Reply:
ma.
which first? if results is an empty list you want to call demisto.results on it (and this is what it won't get into this if)